scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/types/TypeProvider.scala

```diff
@@ -213,6 +213,14 @@ private[types] object TypeProvider {
               tq"_root_.java.lang.String @${typeOf[BigQueryTag]}",
               q"{$rhs}.wkt"
             )
+          case q"$m val $n: _root_.com.spotify.scio.bigquery.types.Json = $rhs" => // Could not find how to mutualize
```

Comment:
I ran some test. This branch is not matched (nor is the Geography one).
I'm wondering if `toTable` works in that case

Reply:
I only tested in real life `fromTable` (that I needed) TBH. But I may need `toTable` in the distant future.

Reply:
I'm trying to adapt the integration test to make sure those types work as expected